app/controllers/admin/generic_events_controller.rb

```diff
@@ -29,7 +29,8 @@ def permitted_params
       pictures_attributes: [:id, :_destroy, :description, :image, :access_level, picture_tag_ids: []],
       team_members_attributes: [:id, :_destroy, :position, :user, :user_id, :proposal],
       attachments_attributes: [:id, :_destroy, :name, :file, :access_level, attachment_tag_ids: []],
-      video_links_attributes: [:id, :_destroy, :name, :link, :access_level, :order]
+      video_links_attributes: [:id, :_destroy, :name, :link, :access_level, :order],
+      reviews_attributes: [:id, :_destroy, :title, :url, :body, :rating, :review_date, :organisation, :reviewer, :event_id],
```

Comment:
Tiny edge case so feel free to ignore but is it worth being able to note if the review was published in print but not online so no URL? Esp. if anyone wants to add archival reviews - guessing currently URL can just be left blank but might be nice to explicitly differentiate between no URL found (yet) and no URL exists